vimms/Controller/misc.py

```diff
@@ -11,140 +8,27 @@ class FixedScansController(Controller):
     A controller which takes a schedule of scans, converts them into tasks in queue
     """
 
-    def __init__(self, ionisation_mode, schedule,
-                 isolation_width=DEFAULT_ISOLATION_WIDTH,
-                 params=None):
+    def __init__(self, schedule, params=None):
```

Comment:
If the user doesn't have a schedule yet (because they need the controller to make an environment), what should they pass here? Empty list, or None? Does it matter?

Reply:
Will change to `schedule=None` instead.

Reply:
might need a change lower (unless `len(None)==0`??)

Reply:
`TypeError: object of type 'NoneType' has no len()`

Reply:
oops ....

Reply:
Done in commit https://github.com/sdrogers/vimms/commit/764e9ce741f22dc2b00f864a49d077e9c7151e82